server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -145,6 +146,9 @@
     public static final Setting<Boolean> DEFAULT_ALLOW_PARTIAL_SEARCH_RESULTS =
             Setting.boolSetting("search.default_allow_partial_results", true, Property.Dynamic, Property.NodeScope);
 
+    public static final Setting<Integer> MAX_OPEN_SCROLL_CONTEXT =
+        Setting.intSetting("search.max_open_scroll_context", 150, 0, Property.Dynamic, Property.NodeScope);
```

Comment:
Let's start at `500` ? That's the number we agreed on in https://github.com/elastic/elasticsearch/pull/29252.

Reply:
Sure, i'll change it while I update the docs